src/commands/force/org/beta/create.ts

```diff
@@ -250,7 +262,104 @@ export class Create extends SfdxCommand {
     }
   }
 
-  private createScratchOrg(): void {
-    //
+  private async saveAuthInfo(username: string): Promise<void> {
```

Comment:
NIT: this method isn't saving the `AuthInfo` of the Org, it could have a more accurate name